internal/mempool/mempool_bench_test.go

```diff
@@ -15,7 +15,11 @@ func BenchmarkTxMempool_CheckTx(b *testing.B) {
 	defer cancel()
 
 	txmp := setup(ctx, b, 10000)
+	txmp.height = 100
```

Comment:
Where did these numbers come from? I have no objection to the change, but could we maybe have a comment nearby to describe why these values vs. something else?

I'm assuming this has to do with making sure we overfill the mempool at appropriate intervals?

Reply:
Was following the setup by https://github.com/tendermint/tendermint/blob/1fe1b6c0327c50ff9e1660e85673665bd9e4d91e/internal/mempool/mempool_test.go#L483

The height and TTLNumBlocks can be removed. The only matter argument will be the mempool size, the cache size and the tx format (with the peerID)